src/publish.js

```diff
@@ -169,7 +173,7 @@ have published it.\
       }
 
       const exists = await this.packageExists(pack.name);
-      if (exists) { return Promise.reject(); }
+      if (exists) { return Promise.resolve(false); }
```

Comment:
This change looks good 👍

<details><summary>Context from looking and the pre-asyncified code (click to expand):</summary>

See the old code here: 

https://github.com/pulsar-edit/ppm/blob/9b7205b04b823fd4835b97fef08c467936541bc2/src/publish.js#L169

We can see that, before the "`async`ification" PR, this function did not throw an error here if `exists` was truthy, rather it just returned the callback without no argument -- i.e. without a truthy argument. Returning `false` here without erroring is fairly equivalent to pre-`async`ification.

---

</details>

**I suppose we need to update the comment above:**

`or resolve with true value` --> `or resolve with true/false value` (or something like that)

Reply:
I suppose it could be an ordinary `return false`. I don't have an opinion on which one is better.